### ISSUE TITLE
[Feature request] Hide some streams from the list and number increment that are not interesting

### ISSUE THREAD

**Issue by r57zone** [Seldszar/Gumbo#143]
Hi, it would be great to hide from the subscription list some streams that are not interesting, and reduce the number of count. 
Thanks for your good plugin.

**Comment by DanzoGit** [comment]
I agree, I would also like to have the option to hide certain streamers as well as some categories.

**Comment by th3r1ko** [comment]
I agree too. It would be a good enhancement

**Comment by Seldszar** [comment]
Version 1.20 allows you to “mute” channels from the dropdown menu of a followed stream. A muted channel will be placed after unmuted ones, will not count in the badge counter, and no notification will be sent.
